runtime/onert/core/include/ir/train/UseDefChains.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __ONERT_IR_TRAIN_USEDEFCHAINS_H__
+#define __ONERT_IR_TRAIN_USEDEFCHAINS_H__
+
+#include "ir/train/UseDefChain.h"
+#include "ir/train/Index.h"
+#include "util/ObjectManager.h"
```

Comment:
Is this header necessary?

Reply:
You are right. It is a legacy left over from revision in the draft.